src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt

```diff
@@ -816,15 +817,15 @@ class RsFunctionStub(
         override fun indexStub(stub: RsFunctionStub, sink: IndexSink) = sink.indexFunction(stub)
     }
 
-    companion object {
-        private val ABSTRACT_MASK: Int =           makeBitMask(RsAttributeOwnerStub.USED_BITS + 0)
-        private val CONST_MASK: Int =              makeBitMask(RsAttributeOwnerStub.USED_BITS + 1)
-        private val UNSAFE_MASK: Int =             makeBitMask(RsAttributeOwnerStub.USED_BITS + 2)
-        private val EXTERN_MASK: Int =             makeBitMask(RsAttributeOwnerStub.USED_BITS + 3)
-        private val VARIADIC_MASK: Int =           makeBitMask(RsAttributeOwnerStub.USED_BITS + 4)
-        private val ASYNC_MASK: Int =              makeBitMask(RsAttributeOwnerStub.USED_BITS + 5)
-        private val HAS_SELF_PARAMETER_MASK: Int = makeBitMask(RsAttributeOwnerStub.USED_BITS + 6)
-        private val IS_PROC_MACRO_DEF: Int =       makeBitMask(RsAttributeOwnerStub.USED_BITS + 7)
+    companion object : BitFlagsBuilder(RsAttributeOwnerStub, INT) {
+        private val ABSTRACT_MASK: Int =           nextBitMask()
```

Comment:
If you decide to use this style, let's
```suggestion
        private val ABSTRACT_MASK: Int           = nextBitMask()
```

Reply:
Also, if you decide to use this style, let's surround this group with `// @formatter:off` and `// @formatter:on` comments to prevent accident formatting of this region